project/src/main/kotlin/org/ionproject/core/common/Uri.kt

```diff
@@ -44,16 +44,20 @@ object Uri {
     const val programmes = "$version/programmes"
     const val programmesById = "$version/programmes/{id}"
     const val programmeOfferById = "$version/programmes/{idProgramme}/offers/{idOffer}"
-    const val programmeByIdOffer = "$version/programmes/{idProgramme}/offers/"
+    const val programmeByIdOffer = "$version/programmes/{id}/offers"
 
     val programmesByIdTemplate = UriTemplate(programmesById)
+
     val programmeOfferByIdTemplate = UriTemplate(programmeOfferById)
-    val programmeByIdOfferTemplate = UriTemplate(programmeByIdOffer)
+    val pagingProgrammes = UriTemplate("${programmes}$rfcPagingQuery")
```

Comment:
Shouldn't we have a constant for `${programmes}$rfcPagingQuery`, assuming it is used on a mapping annotation?

Reply:
This value is only used for URI Templates (present in the json-home resource) whereas the Controller mapping annotations don't use this template. Moreover, I don't think it would work with this query template `{?page,limit}`